lib/stat_tracker.rb

```diff
@@ -33,9 +33,27 @@ def total_goals_per_game
     end
   end
 
+  def total_away_wins
+    away_wins = 0
+    @games.each do |game|
+      if game.away_goals > game.home_goals
+        away_wins += 1
+      end
+    end
+    away_wins
+  end
+
 # Game Statistics Tests - Stat Methods #
   def highest_total_score
-    total_goals_per_game.max_by {|game_id, total_goals| total_goals}[1]
+    total_goals_per_game.max_by {|game_id, total_goals| total_goals}
+  end
+
+  def lowest_total_score
+    total_goals_per_game.min_by {|game_id, total_goals| total_goals}
+  end
+
+  def percentage_visitor_wins
+    (total_away_wins.to_f / @games.count).round(2)
```

Comment:
You should multiply this value by 100 to get a percentage. Again, the test missed this because of assert vs assert_equal

Reply:
Oof. It works now. Those asserts were killer.